meeting_notes/2018-0-28.md

```diff
@@ -0,0 +1,52 @@
+## Participants
+
+- Alois Reitbauer
+- Bogdan Drutu
+- Sergey Kanzhelev
+- Erika Arnold
+- Ted Young
+- Reiley
+- Yuri Shkuro
+
+
+
+## Decision to make specs public
+
+Alois asked whether anyone objects making the specs public. People uniformely agreed to officially share the specs
+
+## Test Suite
+
+Reiley did a demo of the current state of the harness. 
+
+Yuri: How to control the behaviour of the harness? 
+Sergey: We will trigger different URL for different behaviour
+Yuri: Makes sense, but not currently part of PR
+
+Yuri: Can we make it simpler to only use a single language
+Reiley and Sergey: Agreed?
+
+Yuri: CLI should be primary goal as it is easy to integrate into Continuous Build systems. 
+
+Yuri: Best case we use a Docker image to run the tests. 
+
+Yuri: Did we discuss having shared libraries for parsing
+Sergey: So far we did not discuss. Opencensus can be the lib for us
+Alois: Fabian showed a library last time. 
+Bogdan: Where do we put - like an 
+
+*We decided to move all the code over to Node and also provide it as a Docker image* 
```

Comment:
we considering node... @reyang made some digging and it may be the case that python will be better

Reply:
Most famous Node libraries would normalize HTTP request/response headers to a dictionary transparently, which makes it hard for the test harness to issue duplicated headers (e.g. request with two tracestate headers "tracestate: foo=1,bar=2" and "tracestate: foo=2,baz=3"). I will do more research this week, it is very likely that I'll choose Python rather than Node.js.
Docker image - definitely yes.